doc/snipe.txt

```diff
@@ -67,6 +67,15 @@ Dvorak users, e.g.
 >
     let g:snipe_jump_tokens = 'AOEUIDHTNS'
 <
+The jump tokens are highlighted in a configurable color. The
+default colors are #fabd2f for gui mode and 7 (light grey). To
+override these values, specify the the variables (for gui mode and
+terminal mode respectively):
+>
+    let g:snipe_highlight_gui_color = '#dfffaf'
+
```

Comment:
Missing `cterm256`?

Reply:
Yeah I just use the same global for both cterm and cterm256, I can update if you want to have 3 levels of granularity.

Reply:
Yeah that sounds good.